index.html

```diff
@@ -66,67 +66,139 @@
     </header>
     <main>
        <div class="wrapper">
-        <div>
+        <div class="container home">
             <p class="orange">"Hello! Sharing the world"</p>
-            <h1 class="orange">CREATE COMMON GLOBAL SUMMIT 2015</h1>
-            <p>
-                Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged.
+            <h1 class="orange home-title">CREATE COMMON <br>GLOBAL SUMMIT <br>2015</h1>
+            <p class="home-text">
+                Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 2004
             </p>
-            <div>
+            <div class="sub-text">
                 <h2>CREATE COMMON GLOBAL SUMMIT 2015</h2>
                 <p>@2022, SelfDev. All rights reserved</p>
             </div>
         </div>
-        <div>
-            <h3 title="title">Main Program</h3>
-            <div class="grid-container">
-                <div class="grid">
-                    <img src="" alt="">
-                    <h4>Lecture</h4>
-                    <p>
+        <div class="container">
+          <div class="services">
+            <div class="title">
+                <h3>Main Program</h3>
+                <hr class="hr">
+            </div>
+            <div class="service-gird-row">
+                <div class="service-gird-col transparent-bg">
+                    <img src="./images/whiteboard.png" class="service-icon" alt="icon">
+                    <h4 class="orange service-title">Lecture</h4>
+                    <p class="service-desc">
+                        Lorem Ipsum is simply dummy text of the printing and typesetting industry.
+                    </p>
+                </div>
+                <div class="service-gird-col transparent-bg">
+                    <img src="./images/whiteboard.png" class="service-icon" alt="icon">
+                    <h4 class="orange service-title">Lecture</h4>
+                    <p class="service-desc">
+                        Lorem Ipsum is simply dummy text of the printing and typesetting industry.
```

Comment:
- [ ] Kindly align the paragraphs in this program's section in the mobile view as seen in the original design.
- Your design:
![Annotation 2022-05-26 090026](https://user-images.githubusercontent.com/85113567/170447835-170d8e5f-3afa-4ad9-9669-18f718f38256.jpg)
- Required design:
![Annotation 2022-05-26 090008](https://user-images.githubusercontent.com/85113567/170447873-09468c6d-6984-44b1-8862-ee7db7e0c8e0.jpg)